distribution/server/pom.xml

```diff
@@ -192,8 +192,36 @@
           <groupId>io.netty</groupId>
           <artifactId>*</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>io.vertx</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
This tmp fix lgtm. And ideal fix would be changing the io.vertx version in BookKeeper. Please help open an issue/PR in BookKeeper, and add an comments here. So when using the BookKeeper with the fix, we could change this back.

Reply:
Yes, i already send a pull request on bk, refer to: https://github.com/apache/bookkeeper/pull/2410